src/backend/src/modules/configuration/configuration.service.ts

```diff
@@ -0,0 +1,18 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import * as fs from 'fs';
+@Injectable()
+export class ConfigurationService {
+  getFeatures() {
+    try {
+      const filePath = process.env.CONFIG_FILE_PATH;
```

Comment:
I think I'd prefer if this data was loaded into memory in the service constructor. That way, if the config file is misconfigured, the pod won't even start-up, rather than relying on 500s returned to the user which we might not see.

Reply:
Please let me know if you need more specific direction on this @binggaofw